safaribooks.py

```diff
@@ -377,14 +377,17 @@ def __init__(self, args):
             cover_html = self.parse_html(
                 html.fromstring("<div id=\"sbo-rt-content\"><img src=\"Images/{0}\"></div>".format(self.cover)), True
             )
+            cover_css = self.parse_css(
+                html.fromstring("<div id=\"sbo-rt-content\"><img src=\"Images/{0}\"></div>".format(self.cover))
+            )			
```

Comment:
You have a bunch of trailing whitespaces on that line.

Reply:
Fixed, thanks!